dispatch/modules/content/models.py

```diff
@@ -361,6 +366,11 @@ def get_absolute_url(self):
         """
         return "%s%s/%s/" % (settings.BASE_URL, self.section.slug, self.slug)
 
+class ArticleRelation(Model):
+    parent = ForeignKey('Article', related_name='parent_article')
+    article = ForeignKey('Article', related_name='article')
+    count = PositiveIntegerField(default=1)
+
```

Comment:
I think this should be part of a separate PR since it's not related to notifications. Correct me if I'm wrong

Reply:
yeah, this branch was off of the new suggested article branch that rowan started I think

Reply:
I removed the article relation stuff